lnrpc/rpc.proto

```diff
@@ -1652,6 +1724,8 @@ message AddInvoiceResponse {
     payment to the recipient.
     */
     string payment_request = 2 [json_name = "payment_request"];
+
+    string chain = 3 [json_name = "chain"];
```

Comment:
I don't think we need to return `chain` here because the chain is already identifiable from the HRP of the invoice and will have been passed in to the AddInvoice call, presumably.

Reply:
See earlier comment. This it should stay as it is now.